tools/RuniOSCmdlets.ps1

```diff
@@ -0,0 +1,81 @@
+﻿## Powershell commandlet execution for MAM
```

Comment:
@rnikhilkumar This looks like a demo script, that might be used for a presentation. Probably not suitable for the Azure PowerShell project, as Mark pointed out.

Cheers,
Trevor Sullivan
Microsoft MVP: PowerShell